angular/projects/spark-angular/src/lib/components/inputs/Monetary.stories.ts

```diff
@@ -37,12 +38,10 @@ export default {
   parameters: {
     info: `
 ${markdownDocumentationLinkBuilder('input')}
-- The value of this field may contain special characters
-(,) which you may need to remove before submitting the form.
-- Requires Additional Engineering:
-  - Validation
-  - Reformatting to monetary pattern (##.##) - On blur,
-  it should reformat to add commas and decimals to the hundredth place.
+- In order to use angular form validation on these inputs, add a pattern attribute containing a regular expression to the input.
+- Formatting can be turned on by adding the sprkFormatterMonetary directive to the input. Formatting is run on blur to add commas and decimals to the hundredth place.
+- When using sprkFormatterMonatery, fields should always have type="text", NOT type="number". Formatted currency (e.g. $1,234.56) is non-numeric.
```

Comment:
Should be sprkFormatterMonetary. Add backticks to `type="text"` and `type="number"` for code formatting.